airflow/configuration.py

```diff
@@ -1808,4 +1827,5 @@ def __getattr__(name):
 
 conf = initialize_config()
 secrets_backend_list = initialize_secrets_backends()
+auth_manager = initialize_auth_manager()
```

Comment:
This module is already hugely dependent on a lot of other modules and very often creates import cycles. Does this have to be initialized here?

Reply:
The auth manager will be used in a lot of code/modules in Airflow. That's where I think it makes to more sense but I can move it if there is a better location to do so